librz/bin/format/mach0/mach0_defines.h

```diff
@@ -1663,6 +1663,33 @@ static inline void dyld_chained_ptr_arm64e_auth_bind24_read(struct dyld_chained_
 	READ_BITS(dst->auth, 1);
 }
 
+// DYLD_CHAINED_PTR_32
+struct dyld_chained_ptr_32_rebase {
+	ut32 target : 26,
+		next : 5,
+		bind : 1; // == 0
+};
+
+static inline void dyld_chained_ptr_32_rebase_read(struct dyld_chained_ptr_32_rebase *dst, ut32 raw_val) {
+	READ_BITS(dst->target, 26);
+	READ_BITS(dst->next, 5);
+	READ_BITS(dst->bind, 1);
+}
+
+struct dyld_chained_ptr_32_bind {
+	ut32 ordinal : 20,
+		addend : 6,
+		next : 5,
+		bind : 1; // == 1
+};
```

Comment:
isn't better if we avoid these kinda of structures with bitfields since they are not compatible with BE archs?

Reply:
This is only an issue if they are plainly copied over. The `dyld_chained_ptr_32_bind_read()` works on be too (see s390x test success).
Still keeping the structure itself as a bitfield is useful to have a reference for the original layout when reading the code.